packages/rps/src/redux/store.ts

```diff
@@ -54,16 +50,18 @@ function* rootSaga() {
     yield fork(autoOpponent, 'B', client);
   } else {
     yield call([window.channelProvider, 'enable']);
-    yield fork(firebaseInboxListener, client, window.channelProvider.signingAddress);
-
-    yield call(
-      [client, 'approveBudgetAndFund'],
-      '0x8ac7230489e80000', // 10 eth
-      '0x8ac7230489e80000',
-      window.channelProvider.selectedAddress,
-      HUB.signingAddress,
-      HUB.outcomeAddress
-    );
+
+    setupFirebase(client, window.channelProvider.signingAddress);
```

Comment:
I don't know why this was necessary, but I like the change because it does the firebase messaging setup without using a saga (fewer lines, matches web3t pattern). 

Syncing open games is unaffected, still uses a saga.